src/js/components/Search.jsx

```diff
@@ -24,11 +54,16 @@ export class Search extends PureComponent {
           onSearchChange={(data, searchTerm) => (
             this.handleSearchChange(data, searchTerm)
           )}
+          onSearchClear={() => this.handleSearchChange(null)}
         />
         {/* render results table */}
         {this.state.showResults && <ResultsTable
           {...props}
+          changeColumnWidths={(columns, table) => this.handleCalculateColumnWidths(columns, table)}
```

Comment:
You are providing this prop, but I don't see in blueprintjs table or in the sussol table wrapper any use of this `changeColumnWidths`.